portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,38 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/*Global variables pertaining to parallax animation*/
+var lFollowX = 0,
+    lFollowY = 0,
+    lFollowZ = 0,
+    x = 0,
+    y = 0,
+    friction = 1 / 10;
```

Comment:
A couple of thoughts here:

- I think, generally, it's good to avoid global variables. Mutable global state is difficult to reason about and can be a source of many bugs.
- Can these variable names be more descriptive? For instance - I'm not sure what `x` and `y` mean here (or many of the other names).